server/sockets/socket-server.js

```diff
@@ -55,10 +55,42 @@ io.on('connection', (socket) => {
     if (people.length > 1) {
       people.allowed = 1;  // setting var for game started true
       io.in(data.roomname).emit('usersList', usernames)       // emitting usernames list to clients
+      let players = people.length                     //no of players
```

Comment:
This could be `const`. Also, please run linting.

Reply:
sorry forget eslint file into folder done now